app/controllers/verify/sessions_controller.rb

```diff
@@ -16,44 +16,39 @@ def new
     end
 
     def create
-      process_step
-    end
-
-    private
+      result = step.submit
+      analytics.track_event(Analytics::IDV_BASIC_INFO_SUBMITTED, result.to_h)
```

Comment:
since we are tracking analytics here, should we have a spec expectation in the specs for this controller?